test/ecto/changeset/embedded_test.exs

```diff
@@ -726,13 +723,10 @@ defmodule Ecto.Changeset.EmbeddedTest do
       |> Changeset.cast_embed(:posts)
       |> Changeset.add_error(:name, "is invalid")
 
-    errors = Changeset.traverse_errors(changeset, fn
-      {err, opts} ->
-        err
-        |> String.replace("%{count}", to_string(opts[:count]))
-        |> String.upcase()
-      err ->
-        String.upcase(err)
+    errors = Changeset.traverse_errors(changeset, fn {msg, opts} ->
+      msg
+      |> String.replace("%{count}", to_string(opts[:count]))
+      |> String.upcase()
```

Comment:
see that the this custom function (`msg_fun` in traverse_errors docs) is duplicated in many tests and in the docs. 

Perhaps it would be worth it, to supply it as the default one for `traverse_errors/2`? It wouldn't catch all possible cases, but it could be a good starting point. Perhaps it could be just do `String.replace` for all opts (`count` etc) that Ecto ships with?